### ISSUE TITLE
[Package Request] - s3fs-fuse

### ISSUE THREAD

**Issue by filipraus** [amazonlinux/amazon-linux-2022#581]
**What package is missing from Amazon Linux 2023? Please describe and include package name.**
Package description: "s3fs allows Linux, macOS, and FreeBSD to mount an S3 bucket via [FUSE(Filesystem in Userspace)](https://github.com/libfuse/libfuse)." [[https://github.com/s3fs-fuse/s3fs-fuse](https://github.com/s3fs-fuse/s3fs-fuse)]
Package name: s3fs-fuse

**Is this an update to existing package or new package request?**
A new package request.

**Is this package available in Amazon Linux 2? If it is available via external sources such as EPEL, please specify.**
It is available via EPEL in Amazon Linux 2.

**Any additional information you'd like to include. (use-cases, etc)**

**Comment by stewartsmith** [comment]
There's also https://github.com/awslabs/mountpoint-s3 which offers similar functionality.

**Comment by jmcpheters** [comment]
There's also https://www.freekb.net/Article?id=4927 which includes details specifically on how to build for AL2023. IMHO, a pre-built package would be preferable, but until then...

**Comment by jtbr** [comment]
AWS EMR documentation [instruct users to install this package](https://docs.aws.amazon.com/emr/latest/ManagementGuide/emr-studio-magics.html#emr-magics) to support the EMR Studio %mount_workspace_dir magic command. Which won't work in EMR 7.0.0, since it uses Amazon Linux 2023.

**Comment by dbouclier** [comment]
> There's also https://github.com/awslabs/mountpoint-s3 which offers similar functionality.

Yes but mountpoint-s3 have some limitations like https://github.com/awslabs/mountpoint-s3/issues/1174
so it's not a fully valid workaround

**Comment by serbinsh** [comment]
I agree with comments about regarding mountpoint-s3 not being a valid workaround for s3fs-fuse and that it would be great to add the library to al2023.  Is this an issue still on the table for a fix?

**Comment by energyaccountnr1** [comment]
As amazonlinux 2 is deprecated now and mountpoint-s3 has some [bug](https://github.com/awslabs/mountpoint-s3/issues/1861) which breaks stuff for us. We support the approach to have a precompiled s3fs package for amazonlinux 2023 .
